src/java.base/windows/native/libjava/canonicalize_md.c

```diff
@@ -138,7 +138,8 @@ lastErrorReportable()
         || (errval == ERROR_BAD_NET_NAME)
         || (errval == ERROR_ACCESS_DENIED)
         || (errval == ERROR_NETWORK_UNREACHABLE)
-        || (errval == ERROR_NETWORK_ACCESS_DENIED)) {
+        || (errval == ERROR_NETWORK_ACCESS_DENIED)
+        || (errval == ERROR_NO_MORE_FILES)) {
```

Comment:
@bplb Is this the same thing as JDK-8234363? Does this also need to be handled in the FindFirstFile usage in WindowsDirectoryStream?

Reply:
It seems likely. I don't recall having been able to replicate these problems however.

Reply:
I used to duplicate it quite easily on my personal laptop with JDK11 (one laptop with Windows 11, the other with Windows 10). After my laptop system got updated, it was almost impossible to recreate it with my two laptops. To verify this bug, we have to configure exactly the same testing environment as the customer was having.